contrib/ceph/micro-osd.sh

```diff
@@ -0,0 +1,90 @@
+#
+#    Copyright (C) 2013,2014 Loic Dachary <loic@dachary.org>
```

Comment:
@noahdesu @stevvooe @hellertime Thanks, I think I'll (re)write a bootstrap script for circle with ceph-deploy.